internal/service/streamconnection/resource_stream_connection_test.go

```diff
@@ -13,24 +14,44 @@ import (
 
 //go:embed testdata/dummy-ca.pem
 var DummyCACert string
+var (
```

Comment:
nit: add DummyCACert to the var block

Reply:
changed in 6f35f2a4e9b54f4d296ad6e55b085b767039861a